webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -193,14 +193,7 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
                       },
                     },
                   ]}>
-                  <MonacoEditor
-                    height="16vh"
-                    options={{
-                      readOnly: !canModifyCPS,
```

Comment:
there are a few places like this where the old editor had `readOnly` and should be passed to `CodeMirrorEditor`